manuscript/manuscript.Rmd

```diff
@@ -315,17 +314,22 @@ plot_sims(sims_df) / ( plot_policy(policy_df) + ylim(c(0,0.8)) + plot_reward(sim
 
 ```
 
-
+The cumulative reward (utility) realized across 100 stochastic replicates is indistinguishable from that of the optimal policy [Fig 2].
+Nevertheless, comparing the mean state over replicate simulations reveals some differences in the RL strategy, wherein the stock is maintained at a slightly higher-than-optimal biomass.
+Because our state space and action space are sufficiently low-dimensional in this example, we are also able to visualize the policy function directly, and compare to the optimal policy [Fig 2].
+This confirms that quotas tend to be slightly lower than optimal, most notably at larger stock sizes.
 These features highlight a common challenge in the design and training of RL algorithms.
 RL cares only about improving the realized cumulative reward, and may sometimes achieve this in unexpected ways.
-Because these simulations rarely reach stock sizes at or above carrying capacity, these larger stock sizes show a greater deviation from the optimal policy than we observe at lower stock sizes.
-As intuition would suggest, it is possible to devise alternative training schemes which may do a better job sampling the simulation dynamics at high stock levels.
+Because these simulations rarely reach stock sizes at or above carrying capacity, these larger stock sizes show a greater deviation from the optimal policy than we observe at more frequently visited lower stock sizes.
 Training these agents in a variety of alternative contexts can improve there ability to generalize to other scenarios.
 
-
-<!-- 
-Add text about empirical example
--->
+How would an RL agent be applied to empirical data?
+In principle, this is straight forward.
+Once our agent has been trained on appropriate simulation environments, it is able to propose a quota given an observation of the stock after appropriate transformation of variables.
```

Comment:
Did we talk about state space transformation above? I feel like we could get away with leaving the transformation bit out and let the motivated see it in appendix B.